src/main.tsx

```diff
@@ -72,6 +73,10 @@ const router = createHashRouter([
         path: 'basket',
         element: <BasketPage />,
       },
+      {
+        path: 'detailPage',
+        element: <DetailPage />,
```

Comment:
We already have a component for detailed product information called `<ProductItemPage />`,  so it seems there was no need to create a new component